src/Toolkit/V.php

```diff
@@ -248,6 +248,21 @@ public static function __callStatic(string $method, array $arguments): bool
                 $date['warning_count'] === 0;
     },
 
+    /**
+     * Parses the first two parameters as dates and then
+     * validates them as specified by the third parameter
+     */
+    'dates' => function ($value, $test, $validator): bool {
```

Comment:
The name is very similar to `date`, which just validates a date value. Also the name doesn't really tell what the validator does (could also be a validator that checks that each value in an array is a valid date).

Also a few minor other suggestions:

```suggestion
    /**
     * Compares the two provided values as dates in any format
     * PHP can parse using the provided comparison operator
     */
    'dateCompare' => function ($value, $test, $operator): bool {
```

I'm not 100 % happy with my name suggestion either as the other validators have names such as `startsWith`, so this validator would need to be called something like `dateComparesAgainst`, but that's long.

Reply:
I know you like very explicit method names, which I don't always see really fitting the overall naming style of Kirby, e.g. `dates` I think is very much in line with `V::errors`. What about `assertDates`?

Also, I don't think we should use `$operator` as what needs to be passed should be a name of another validator method - I would keep it consistent what they are called within the `V` class.

Reply:
I like the idea!

What about `assertDate`? The validator validates that the `$value` validates against an assertion with another date. Together these are two dates, yes, but all other validators are named from the perspective of `$value`. So I think `assertDate` would fit quite well.